app/components/doi-metrics-badge.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+
+export default Component.extend({
+  tagName: 'div',
+  classNames: [ 'license' ],
+  citations: 0,
+  views: 0,
+  downloads: 0,
+  label: '',
+  metrics: [],
+  store: service(),
+  hasUsage: computed('views','downloads', function() {
+    return ((this.views + this.downloads) > 0) ?  true  : false;
+  }),
+  hasMetrics: computed('citations','views','downloads', function() {
+    return ((this.views + this.downloads + this.citations) > 0) ?  true  : false;
+  }),
+
+  init() {
```

Comment:
Why are you writing this, it is the default behaviour.

Reply:
doi-metrics-badge.js  has been removed